spec.md

```diff
@@ -8,66 +8,96 @@ title: Semantic Specification
 
 
 ~~~
-Parentage/causal relationships for spans from a single trace
+Causal relationships for Spans in a single trace
 
-        [Span A]  <--(the root span)
+        [Span A]  ←←←(the root span)
             |
      +------+------+
      |             |
- [Span B]      [Span C]
+ [Span B]      [Span C] ←←←(Span A is Span C's parent)
      |             |
- [Span D]      +---+-----+
-               |         |
-           [Span E]  [Span F]
-                         |
-                +--------+--------+
-                |        |        |
-            [Span G] [Span H] [Span I]
+ [Span D]      +---+-------+
+               |           |
+           [Span E]    [Span F] >>> [Span G] >>> [Span H]
```

Comment:
where can I find out more about "sequence predecessors"? I don't see it in the use cases doc although I can grok it. Particularly interested in how folks use the OT to annotate the sequence relationship. I'm also interested in how folks represent this in implementation. For example, I know Naver use a "nextId" concept to help, although depending on the semantics of sequence predecessor, it may not be the same thing.

https://github.com/naver/pinpoint/blob/d42fb8437a4098ed375790810c5a30452f05074d/bootstrap-core/src/main/java/com/navercorp/pinpoint/bootstrap/context/AsyncTraceId.java#L11

cc @xylus